shim/linux/src/entry.c

```diff
@@ -572,7 +574,7 @@ static long
 dispatch_vm_kvm_set_identity_map_addr(uint64_t *const ioctl_args)
 {
     (void)ioctl_args;
-    return -EINVAL;
+    return 0;
```

Comment:
I see we start returning success for these now, what are the side effects of returning success to userspace when there is no implementation? Perhaps adding a TODO describing the situation would be useful here.